ui/src/containers/VisualiseForm/StatementsForm/AxesEditor/CourseEditor.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component, PropTypes } from 'react';
```

Comment:
```suggestion
import React, { Component } from 'react';
import PropTypes from 'prop-types';
```

Reply:
?

Reply:
https://reactjs.org/docs/typechecking-with-proptypes.html

> React.PropTypes has moved into a different package since React v15.5. Please use the prop-types library instead.

We've already upgraded react to v16. Importing `PropTypes` from "react" will not work in master branch.